falcosidekick/Chart.yaml

```
@@ -1,9 +1,9 @@
 apiVersion: v1
-appVersion: 2.23.1
+appVersion: 2.24.0
 description: A simple daemon to help you with falco's outputs
```

```suggestion
description: Connect Falco to your ecosystem
```

I prefer this description now

---

fixed